wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -281,6 +284,17 @@ impl PhysicalDeviceFeatures {
             } else {
                 None
             },
+            depth_clip_enable: if enabled_extensions.contains(&vk::ExtRobustness2Fn::name()) {
```

Comment:
why does it depend on the `VK_EXT_robustness2` extension?

Reply:
copy-paste error :sweat: